root/etc/cont-init.d/50-config

```diff
@@ -81,6 +81,12 @@ cp /config/fail2ban/jail.local /etc/fail2ban/jail.local
 [[ ! -f /config/www/502.html ]] &&
 	cp /defaults/502.html /config/www/502.html
 
+# remove lua bits from nginx.conf if not done before
+if ! grep -q '#Removed lua' /config/nginx/nginx.conf; then 
+  echo "Removing lua specific info from nginx.conf"
+  sed -i 's|\tlua_load_resty_core off;|\t#Removed lua. Do not remove this comment|g' /config/nginx/nginx.conf
+fi
```

Comment:
```suggestion
# remove lua bits from nginx.conf if not done before
if grep -q 'lua_load_resty_core' /config/nginx/nginx.conf; then 
  echo "Removing lua specific info from nginx.conf"
  sed -i '/lua_load_resty_core/d' /config/nginx/nginx.conf
fi
```
Let's just delete any line containing `lua_load_resty_core` ? In which case the persistent comment is not required.

Reply:
if the user wants to install lua manually or via a mod, and needs to disable resty core, this would prevent them from doing so by forcefully removing it during each start

Reply:
we only want to remove it once, and replace it with a custom message. If they add it again, it will stay as long as our custom message is there, confirming that we had removed it that one time

Reply:
Fair enough.